ex03.exs

```diff
@@ -52,11 +52,56 @@ defmodule Ex03 do
 
   def pmap(collection, process_count, function) do
```

Comment:
As you said in your commit message, you are close. But, at the same time, you've c=gone for a fairly procedural
style of coding, using `for` comprehensions where simply maps would do. 

It would be a lot easier to give the processes the work to do when you spawn them, and then just wait dfor each to finish in turn.

# 15